server/main.lua

```diff
@@ -16,12 +16,20 @@ local playerState
 
 AddEventHandler('QBCore:Server:OnPlayerLoaded', function()
 	playerState = Player(source).state
+	playerState:set(DEATH_STATE_STATE_BAG, Config.DeathState.ALIVE, true)
```

Comment:
Wouldn't we apply if the player was dead via the PlayerData from core here?

Reply:
good point. Fixed to load from metadata.